pkgs/development/tools/yarn2nix-moretea/yarn2nix/default.nix

```diff
@@ -413,7 +413,7 @@ in rec {
     # we import package.json from the unfiltered source
     packageJSON = ./package.json;
 
-    yarnFlags = defaultYarnFlags ++ ["--production=true"];
+    yarnFlags = defaultYarnFlags ++ [ "--ignore-scripts" "--production=true" ];
```

Comment:
This shouldn't be necessary since you default `ignoreScripts` to true.

Reply:
This is a different function which does not take any arguments.

Reply:
Isn't it the same? This function is `mkYarnPackage` which forwards `yarnFlags` to `mkYarnModules`?